static/js/zamboni/files.js

```diff
@@ -112,6 +112,32 @@ var Highlighter = {
         // highlighter will try to replace the original element in the
         // DOM when it's done.
 
+        // Verify that we actually support the used brush
+        // see https://github.com/mozilla/addons-server/issues/4552 for more
+        // details. This shows an alert if a brush is not supported.
+        var discoveredBrushes = SyntaxHighlighter.brushes;
+
+        if (discoveredBrushes) {
+            var brushes = [];
+
+            for (var discoveredBrush in discoveredBrushes) {
```

Comment:
We actually iterate only over non-integer properties in the array, unless I misunderstand you.